### ISSUE TITLE
FATAL: 1.1.7  to 1.2.0-beta.6 breaks fullscreen mode ('active' of undefined)

### ISSUE THREAD

**Issue by TheOneAndOnlyGarcia** [jossmac/react-images#396]
**Steps to reproduce the behavior:**
 upgrade from 1.1.7  to 1.2.0-beta.6

tested with react 16 and 17!

**Expected behavior:**
open fullscreen should not throw an error

**Actual behavior:**
opening fullscreen throws error

`
Error in function FullScreen in ./node_modules/react-full-screen/dist/index.modern.js:56
`
🚨 **Cannot read property 'active' of undefined** 🚨

```js
  56 |   if (handle.active) {
  57 |     classNames.push('fullscreen-enabled');
  58 |   }
```

**Comment by gtwilliams03** [comment]
I am getting the same error now.  But I think the error is coming from the `FullScreen` component of `react-fullscreen`.

**Comment by gtwilliams03** [comment]
I am not sure how to do a pull request, but I think the error is in line 193 of the `/Modal/Modal.js` file.

This:

```
<FullScreen enabled={isFullscreen} onChange={this.handleFullscreenChange}>
```

should be:

```
<FullScreen enabled={isFullscreen} onChange={this.handleFullscreenChange} handle={_handle}>
```

like in their examples:

```
import { FullScreen, useFullScreenHandle } from 'react-full-screen'
...
const _handle = useFullScreenHandle()
```

**Comment by gtwilliams03** [comment]
Never mind - we just need to wait for the pull request at #397 from @foglerek to get merged.  It looks like he has this sorted already.

**Comment by luskin** [comment]
Bump. Can we get this merged in?

**Comment by davwheat** [comment]
Done. Releasing now! Sorry all -- been stressing with some exams coming up.

**Comment by luskin** [comment]
Appreciate you @davwheat, thanks for jumping in and getting this merged 🙌🏼

**Comment by davwheat** [comment]
Released: `1.2.0-beta.7`
